autofill-parser/src/main/java/com/github/androidpasswordstore/autofillparser/FeatureAndTrustDetection.kt

```diff
@@ -81,7 +81,8 @@ private val TRUSTED_BROWSER_CERTIFICATE_HASH =
     "org.ungoogled.chromium.extensions.stable" to
       arrayOf("29UOO5cXoxO/e/hH3hOu6bbtg1My4tK6Eik2Ym5Krtk="),
     "com.kiwibrowser.browser" to arrayOf("wGnqlmMy6R4KDDzFd+b1Cf49ndr3AVrQxcXvj9o/hig="),
-    "us.spotco.fennec_dos" to arrayOf("/4H1vlY5ZZTu5w/vKDIlbhUhQSLiupzt0mAF/9S8qqg="),
+    "us.spotco.fennec_dos" to arrayOf("Jg4KSWeMeLcMAtZTet07bcChcXG73oznX9QCaoo+GNI="),
+    "us.spotco.fennec_dos_fdroid" to arrayOf("/4H1vlY5ZZTu5w/vKDIlbhUhQSLiupzt0mAF/9S8qqg="),
```

Comment:
```suggestion
    "us.spotco.fennec_dos_fdroid" to arrayOf("/4H1vlY5ZZTu5w/vKDIlbhUhQSLiupzt0mAF/9S8qqg="),
```

This package name is wrong, F-Droid's Mull browser is packaged as [`us.spotco.fennec_dos`](https://f-droid.org/en/packages/us.spotco.fennec_dos/) too.

Reply:
You need to add the certificate hash like [this](https://github.com/android-password-store/Android-Password-Store/pull/1166/commits/96787311a0b5a3d412ddf53c26e5cba769e2d72e) as an alternative hash for the same package rather than create a fake one.

Reply:
Thank you! I think it's good now. https://github.com/android-password-store/Android-Password-Store/commit/290c43311a3e4133edbb2175cd3c478de47ee331